### DISCUSSION TITLE
Generate password hash for flood via command line

### DISCUSSION THREAD

**Discussion by SSoft7** [jesec/flood#419] [General]
Hello,

I'm currently using the latest compiled binary and I'm trying to automate the user creation. The problem seems to be with my password generation command.

I've translated this [code](https://github.com/jesec/flood/blob/32fde2cf1a7b9014c59c39a635c815830b7e249f/server/models/Users.ts#L15)  from your codebase to the following command
`echo password | argon2 salt-12345678910 -id -p 1 -t 256 -k 512 -l 32 -e` . 

However, after using the generated hash in the json db, I get `Failed login.`  Can you tell me If I'm using any wrong parameters to generate the hash?

Thanks in advance.

**Comment by SSoft7** [comment]
This is solved. Please close/delete this discussion. I can't seem to delete/close it myself.

**Comment by jesec** [comment]
You can also use Flood's `auth` APIs to create users. 

https://flood-api.netlify.app/#operation/auth.register
